server.js

```diff
@@ -55,208 +56,229 @@ function showHomepage(req, res) {
           //then render the page
           res.render('pages/index', { cities: results.rows, artists: artistsResult.rows });
         })
-        .catch(error => handleErrors(error,res));
+        .catch(error => handleErrors(error, res));
     });
 }
 
 function showArtwork(req, res) {
-  let sql = `SELECT * FROM artworks WHERE artist=$1;`;
+  let sql = `SELECT  FROM artworks WHERE artist=$1;`;
```

Comment:
broken code

Reply:
This should look like this: 
SELECT id, artist AS artistName, city, museum, title as artworkTitle, description AS artworkDescription, image as artworkImage FROM artworks WHERE artist=$1;

Reply:
fixed.